plotmodel.py

```diff
@@ -164,6 +168,17 @@ def makePlot(self):
 
         # set model image
         self.image = image
+        # set model properties
+        self.properties = props
+
+        self.properties[self.properties < 0.0] = 0.0
```

Comment:
I think matplotlib's convention for masking invalid values is to set them to `np.nan`. Would this work here?

Reply:
I think the real issue here is that `get_property_map` is returning non-physical values representing either `VOID` or `NOT_FOUND` that have to be masked out. These should probably be 0.0 coming out of the capi to begin with...

Reply:
Was just looking at this again and I definitely missed what you were getting at. Yes, `np.nan` works just fine here. 

Still curious to see what you think about what `get_property_map` should return for `NOT_FOUND` regions though!